@here/harp-webtile-datasource/lib/HereWebTileDataSource.ts

```diff
@@ -29,7 +30,7 @@ const textureLoader = new TextureLoader();
 /**
  * Options for {@link HereWebTileDataSource}.
  */
-interface HereWebTileDataSourceOptions {
+interface HereWebTileDataSourceOptions extends Omit<WebTileDataSourceOptions, "dataProvider"> {
```

Comment:
you can remove `resolution` and `renderingOptions`, because they are part of `WebTileDataSourceOptions`

Reply:
done